cxxheaderparser/_ply/lex.py

```diff
@@ -233,9 +233,9 @@ def token(self):
                     if tok.type:
                         self.lexpos = m.end()
                         return tok
-                    else:
```

Comment:
Please remove any of the `if`/`elif`/`else` + return related fixes. I think explicit `if`/`elif`/`else` often reads better than the alternative. Everything else seems fine.

Reply:
Done in commits https://github.com/robotpy/cxxheaderparser/pull/71/commits/059ec3e1656125df6093a24d306b2480aba733e9 and https://github.com/robotpy/cxxheaderparser/pull/71/commits/ea4c7edf9930cb939d2b4104d503b24ca64fe5dd